src/Libraries/RevitNodes/Elements/Views/ScheduleView.cs

```diff
@@ -0,0 +1,256 @@
+﻿using RevitServices.Persistence;
+using Autodesk.DesignScript.Runtime;
+using View = Revit.Elements.Views.View;
+using RevitServices.Transactions;
+using DynamoServices;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace Revit.Elements.Views
+{
+    /// <summary>
+    /// Base class for Revit Plan views
```

Comment:
I somehow managed to screw up my branches and pushed this commit into a wrong PR. My bad. This PR has more stuff in it than just Views and ElementType